tests/test_ass.py

```diff
@@ -122,3 +122,19 @@ def test_custom_line_section_write(self, line_section):
 
     def test_custom_line_section_dump(self, line_section):
         assert "\n".join(line_section.dump()) == self.TEST_CUSTOM
+
+
+class TestEvents:
+
+    def test_int_value(self):
+        # Some scripts use empty margins for 0 and floating point numbers.
+        script = dedent("""\
+            [Events]
+            Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+            Dialogue: 0,0:00:02.59,0:00:03.88,TopLeft,,,,25.00,,Some text.
```

Comment:
Also create a test for signed values, and possibly leading space? e.g.
```
Dialogue: 0,0:00:00.00,0:00:05.00,Default,,+514,-494,  33,,Sample line
```
should have margin values of ``514``, ``-494``, and ``33``, while
```
Dialogue: 0,0:00:00.00,0:00:05.00,Default,,-24,- 44,+ 72,,Sample line
```
should have values `-24`, `0`, `0`